docs/storage/storage.md

```diff
@@ -180,7 +180,7 @@ correct version to request when reading nodes.
 #### Reading Leaves
 
 API requests for leaf data involve a straightforward query by leaf data hash, leaf Merkle hash or
-leaf index followed by formatting and marshalling the data to be returned to the client.
+leaf index followed by formatting and marshaling the data to be returned to the client.
```

Comment:
Thanks for the spelling explanation.

I don't have strong feelings in regards to which locale to use. Let's talk to the team and see what they think.